common/src/main/java/us/myles/ViaVersion/api/data/UserConnection.java

```diff
@@ -246,4 +247,77 @@ public void sendRawPacketToServer(final ByteBuf packet, boolean currentThread) {
     public void sendRawPacketToServer(ByteBuf packet) {
         sendRawPacketToServer(packet, false);
     }
+
+    /**
+     * Monitors serverbound packets
+     *
+     * @return true if this packet should be cancelled
+     */
+    public boolean checkIncomingPacket() {
```

Comment:
Semantically, I would expect a checkX method to return true if the content is alright, but maybe that's just me 👀